src/index.ts

```diff
@@ -82,3 +85,28 @@ export const parseKind = (kind: string, { rootSeparator, groupSeparator }: Separ
     groups,
   };
 };
+
+/**
+ * csf story definition
+ */
+export interface CSFStory {
+  story?: {
+    /**
+     * story name if differnet from the export name
+     */
+    name?: string;
+
+    /**
+     * optional collection of properties, which values
+     * will be passed onto the story function
+     */
+    properties?: StoryProperties;
+
+    /**
+     * optional collection of story parameters
+     */
+    parameters?: {
+      [key: string]: any;
+    };
+  };
+}
```

Comment:
missing decorators

Reply:
aren't decorators specific to StoryBook?

Reply:
no i think they are part of CSF. open to discussion on this, i haven't thought about it much.

Reply:
ok to leave decorators out of the scope of this PR. 
They would involve also StoryFn and Context types